datadog/resource_datadog_service_level_objective_test.go

```diff
@@ -136,6 +150,12 @@ func TestAccDatadogServiceLevelObjective_Basic(t *testing.T) {
 						"datadog_service_level_objective.foo", "thresholds.1.timeframe", "30d"),
 					resource.TestCheckResourceAttr(
 						"datadog_service_level_objective.foo", "thresholds.1.target", "98"),
+					resource.TestCheckResourceAttr(
+						"datadog_service_level_objective.foo", "thresholds.1.warning", "99.0"),
```

Comment:
One last thing, this needs to be changed from `99.0` to `99` (because of how TF works internally). Otherwise this LGTM, so once you fix this, I can merge. Thanks @mzneos!

Reply:
Oh ok, my bad. I added this check as it was missing.
I should have run the acceptance tests by myself to make sure everything was working, but wasn't really sure of how to properly do it, sorry for that.

It is fixed now